### PR TITLE
[IGNORE] add project documentation

### PR DESCRIPTION
**Pull request by Nexucis**

### CODE REVIEW

File: docs/project.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Project
+
+A project can be defined as a workspace. It's the place where user will spend most of their time, creating dashboard and
```
Comment (Gladorme) [inline docs/project.md:3]:
```suggestion
A project can be defined as a workspace. It's the place where user will spend most of their time, creating dashboards, variables and
```

File: docs/project.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+datasource.
```
Comment (Gladorme) [inline docs/project.md:4]:
```suggestion
datasources.
```

File: docs/project.md (hunk 1 above)
Comment (AntoineThebaud) [reply docs/project.md:3]: +user**s**

File: docs/project.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+**You need to own a project if you want to create a dashboard**
+
+Creating a project is simple as providing the name:
```
Comment (AntoineThebaud) [inline docs/project.md:8]:
```suggestion
Creating a project is as simple as providing its name:
```

**Pull request merged**